plugin/ctrlsf.vim

```diff
@@ -48,7 +48,7 @@ endf
 " s:SearchCwordCmd() {{{2
 func! s:SearchCwordCmd(type, to_exec)
     let cmd = ":\<C-U>" . a:type . " " . expand('<cword>')
-    let cmd .= a:to_exec ? "<CR>" : " "
```

Comment:
http://blog.csdn.net/boyinnju/article/details/6877087
I am not sure,if we can use `<CR>` instead.